platform/ios/app/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,8 @@
+
+/*
+  File.strings
```

Comment:
Move this file to Base.lproj. A base localization will be reused by all localizations without the need to a separate stub file for each localization.